tests/mocks/account_retriever.go

```diff
@@ -0,0 +1,65 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
This is unintended, looks like mocks need to be updated to prevent this file from generating

Reply:
It should get generated when you run `mockgen` afaik